app/models/iiif_presentation_v3.rb

```diff
@@ -144,10 +144,11 @@ def extract_value(field, hash)
   # rubocop:enable Metrics/PerceivedComplexity
 
   def sanitize_and_wrap(value)
+    # byebug
```

Comment:
comment

Reply:
Thank you for catching this!  Removed that comment.